pkg/workloadmeta/collectors/internal/containerd/container_builder.go

```diff
@@ -38,6 +40,25 @@ func buildWorkloadMetaContainer(namespace string, container containerd.Container
 		log.Debugf("cannot split image name %q: %s", info.Image, err)
 	}
 
+	// Prepare context
+	ctx := context.Background()
+	ctx = namespaces.WithNamespace(ctx, namespace)
+
+	// Get image id from container's image config
+	img, err := container.Image(ctx)
+	if err != nil {
+		log.Debugf("cannot get container's image: %s", err)
```

Comment:
Are there some common cases where it’s more or less expected to not be able to get the image from the container ?
If so, we should keep the `log.Debugf`. But if not, we may want a more verbose log level.

Reply:
There is not a particular reason why it would fail to get the image. So I'll raise it to a "warn"